portfolio/src/main/webapp/script.js

```diff
@@ -118,3 +107,125 @@ function authentication(){
     }
     });
 }
+
+
+var map;
```

Comment:
Usually global variables is a bad practice. For example, the noComm variable above, you can pull from html.
For this parameters, why can't we create a class with members map and editMarker?
Am I right that editMarker is the marker, that is currently edited? Then editedMarker, currentMarker sounds a bit better, because now it sounds like a boolean whether marker is edited or not, or even a function name :)

Reply:
Yes, that is right. Great! Then I will make the changes. Thank you